libs/ngrx-traits/signals/src/lib/with-sync-to-route-query-params/with-sync-to-route-query-params.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { computed } from '@angular/core';
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { ActivatedRoute, provideRouter, Router } from '@angular/router';
+import { withSyncToRouteQueryParams } from '@ngrx-traits/signals';
+import { patchState, signalStore, withState } from '@ngrx/signals';
+import { of } from 'rxjs';
+
+describe('withSyncToRouteQueryParams', () => {
```

Comment:
Hey @gabrielguerrero would you mind adding some tests for the trait with collection?